scripts/deploy_subnet_under_calibration_net/prepare_local.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# A script which resets your local .ipc folder so we can run the deploy.sh script for a local deployment
+#
+# To run:
+# 1. Run `./prepare_local.sh`
+# 2. Run `./deploy.sh`
+#
+# Known issues:
+# - The deploy.sh script does not run "make docker-build" which can cause errors if its outdated on your
+#    machine. To fix this, run `make docker-build` in ipc/fendermint folder and rerun the deploy.sh script
+
+set -eo pipefail
+
+SCRIPT_DIR=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+echo "SCRIPT_DIR: $SCRIPT_DIR"
+
+echo "Installing dependencies..."
+cargo install toml-cli
+
+echo "Removing existing .ipc folder"
+rm -rf ~/.ipc
+
+echo "Copying new .ipc folder"
+cp -r $SCRIPT_DIR/.ipc ~/.ipc
+
+echo "Fetching info about newest current deployment contracts"
+hash=$(curl -s https://raw.githubusercontent.com/consensus-shipyard/ipc/cd/contracts/deployments/r314159.json)
+echo $hash | jq
+
+echo "Parsing gateway and registry address..."
+gateway_addr=$(echo $hash | jq -r '.gateway_addr')
+echo "- Gateway address: $gateway_addr"
+registry_addr=$(echo $hash | jq -r '.registry_addr')
+echo "- Registry address: $registry_addr"
+
+echo "Updating config with new gateway and registry address"
+toml set ~/.ipc/config.toml subnets[0].config.gateway_addr $gateway_addr > ~/.ipc/config.toml.tmp
+toml set ~/.ipc/config.toml.tmp subnets[0].config.registry_addr $registry_addr > ~/.ipc/config.toml.tmp2
+cp ~/.ipc/config.toml.tmp2 ~/.ipc/config.toml
```

Comment:
Can it not do an in-place update, or overwrite itself? It seems error prone to have to create a new file in each step. In any case, do they have to stick around?

Reply:
I believe toml-cli only outputs the modified toml version, see https://github.com/gnprice/toml-cli?tab=readme-ov-file#toml-set. But yeah can cleanup the temp files.